test/hotspot/jtreg/vmTestbase/nsk/jvmti/RedefineClasses/StressRedefine/TestDescription.java

```diff
@@ -36,5 +36,6 @@
  *      -agentlib:stressRedefine
  *      nsk.jvmti.RedefineClasses.StressRedefine
  *      ./bin
+ *      -redefiningThreadsNumber 4
```

Comment:
So `-redefiningThreadsNumber` has always been an allowed option to `StressRedefine`, yet this is the first time it's ever been used? Not an issue. Just a bit odd that the option was already there for you to use.

Reply:
The StressRedefine class seems to be designed as a base for more advanced stress testing. My guess is there was a plan to add more test cases to it but it did not happen.